### PR TITLE
fix(controls): Add aria-expanded to sidebar toggle button

### PR DESCRIPTION
**Pull request by hormaza**
The button controls a collapsible section of content and should have an aria-expandedattribute that reflects its current state (i.e., "true" when expanded and "false" when collapsed)

### CODE REVIEW

**Comment by hormaza** [comment]
image

Here a screenshot of how aria-expanded react to the expansion or contraction of thumbnail sidebar

File: src/lib/viewers/doc/DocBaseViewer.js [hunk 1]
```diff
@@ -1097,6 +1097,7 @@ class DocBaseViewer extends BaseViewer {
                 pageCount={this.pdfViewer.pagesCount}
                 pageNumber={this.pdfViewer.currentPageNumber}
                 scale={this.pdfViewer.currentScale}
+                thumbnailsSidebarIsOpen={this.thumbnailsSidebar ? this.thumbnailsSidebar.isOpen : undefined}
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocBaseViewer.js:1100]: It seems like our default value should be `false` rather than `undefined` here?

File: src/lib/viewers/doc/DocControls.tsx [hunk 2]
```diff
@@ -40,12 +40,16 @@ export default function DocControls({
     pageCount,
     pageNumber,
     scale,
+    thumbnailsSidebarIsOpen,
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocControls.tsx:43]: Let's use `isThumbnailsOpen` to stay consistent with the callback naming?

**Review by jstoffan — COMMENTED** [review]
Thanks for submitting this PR. Please add unit tests to cover both the positive and negative cases.

File: src/lib/viewers/doc/DocBaseViewer.js (hunk 1 above)
Comment (hormaza) [reply src/lib/viewers/doc/DocBaseViewer.js:1100]: updated!

File: src/lib/viewers/doc/DocControls.tsx (hunk 2 above)
Comment (hormaza) [reply src/lib/viewers/doc/DocControls.tsx:43]: updated!

File: src/lib/viewers/controls/sidebar/ThumbnailsToggle.tsx [hunk 3]
```diff
@@ -4,15 +4,17 @@ import './ThumbnailsToggle.scss';
 
 export type Props = {
     onThumbnailsToggle?: () => void;
+    isThumbnailsOpen?: boolean;
```
Comment (jstoffan) [inline src/lib/viewers/controls/sidebar/ThumbnailsToggle.tsx:7]: We typically try to keep our props, types, variables, etc. alphabetized when possible.

File: src/lib/viewers/doc/DocBaseViewer.js [hunk 4]
```diff
@@ -1097,6 +1097,7 @@ class DocBaseViewer extends BaseViewer {
                 pageCount={this.pdfViewer.pagesCount}
                 pageNumber={this.pdfViewer.currentPageNumber}
                 scale={this.pdfViewer.currentScale}
+                thumbnailsSidebarIsOpen={this.thumbnailsSidebar && this.thumbnailsSidebar.isOpen}
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocBaseViewer.js:1100]: Should this be renamed, as well?

File: src/lib/viewers/controls/sidebar/ThumbnailsToggle.tsx (hunk 3 above)
Comment (hormaza) [reply src/lib/viewers/controls/sidebar/ThumbnailsToggle.tsx:6]: Updated!

File: src/lib/viewers/doc/DocBaseViewer.js (hunk 4 above)
Comment (hormaza) [reply src/lib/viewers/doc/DocBaseViewer.js]: Yes. Updated

File: src/lib/viewers/doc/DocControls.tsx [hunk 5]
```diff
@@ -40,12 +40,13 @@ export default function DocControls({
     pageCount,
     pageNumber,
     scale,
+    isThumbnailsOpen,
```
Comment (jstoffan) [inline src/lib/viewers/doc/DocControls.tsx:43]: Since we still need to add tests, can we alphabetize the props here and in ThumbnailsToggle, while we're at it?

Comment (hormaza) [reply src/lib/viewers/doc/DocControls.tsx]: yes! Totally

**Review by jstoffan — APPROVED** [review]
